http/1.10/modules/ROOT/pages/http-request-ref.adoc

```diff
@@ -511,7 +511,7 @@ If you configure this system property, the HTTP *Request* operation starts sendi
 
 Configure this system property when starting Mule by adding: +
  `./mule -M-Dmule.http.requestStreaming.enable=true` +
- This property defaults to `false`.
+ The default value of this property depends on the Mule runtime version. For Mule 4.4 and lower, the property defaults to `false`. For Mule 4.5 and later, the property defaults to `true`.
```

Comment:
I think our standard is to say earlier and later for version, so it would be Mule 4.4 and earlier. This might have changed and I can't find the page now.